types/ffprobe/ffprobe-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as ffprobe from 'ffprobe';
+
+(async () => {
+    const result = await ffprobe('/path/to/movie.avi');
+    result; // $ExpectType FFProbeResult
+})();
```

Comment:
```suggestion
import ffprobe = require('ffprobe');
import ffprobeStatic = require('ffprobe-static');

ffprobe('./file.mp4', { path: ffprobeStatic.path })
    .then(info => {
        info; // $ExpectType FFProbeResult
    })
    .catch((err: Error) => {
        err; // $ExpectType Error
    });

ffprobe('./file.mp4', { path: ffprobeStatic.path }, (err, info) => {
    if (err) {
        err; // $ExpectType Error
    } else {
        info; // $ExpectType FFProbeResult
    }
});

(async () => {
    const result = await ffprobe('/path/to/movie.avi', { path: ffprobeStatic.path });
})();
```
AFAIK this reflects changes in method signature to support 2 required params (`path` and `options`) and support for both Promise<info> and `callback(err, info)` as per package public documentation:
https://github.com/eugeneware/ffprobe#ffprobemediafilepath-opts-cb